### PR TITLE
Get students

### PR DESCRIPTION
**Pull request by david-ron**
we need students for front end

### CODE REVIEW

**Review by bit172 — APPROVED** [review]
Looks good. Bet

**Review by leojrsilao — APPROVED** [review]
bet

File: src/main/java/com/kimsquitsystem/kimsquitdemo/dao/StudentDao.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            String lastName = resultSet.getString("last_name");
+            String address = resultSet.getString("address");
+            int group_id = resultSet.getInt("group_id");
+            return new Student(user_id, firstName,lastName, address, group_id);
+        });
+        return students;
+    }
+
+    public Optional<Student> selectStudentById(String id) {
+        final String sql = "SELECT student_id, first_name, last_name, address, group_id  FROM student WHERE user_id = ?";
```
Comment (bit172) [inline src/main/java/com/kimsquitsystem/kimsquitdemo/dao/StudentDao.java]: student_id not user_id

**Review by bit172 — CHANGES_REQUESTED** [review]

File: src/main/java/com/kimsquitsystem/kimsquitdemo/dao/StudentDao.java (hunk 1 above)
Comment (david-ron) [reply src/main/java/com/kimsquitsystem/kimsquitdemo/dao/StudentDao.java]: okk

Comment (david-ron) [reply src/main/java/com/kimsquitsystem/kimsquitdemo/dao/StudentDao.java]: done

**Review by bit172 — APPROVED** [review]

**Comment by bit172** [comment]
#8

**Pull request merged**